build/tslint.json

```diff
@@ -72,14 +84,18 @@
       "avoid-escape"
     ],
     "radix": true,
-    "semicolon": [true, "always"],
+    "semicolon": [false, "always"],
```

Comment:
Same as above, I see errors if not set it to false. Please try 'gulp tslint' and compare with true/false. I would like to set this one true but I am not sure it's about time to change code to fix the errors.